navigator/src/main/java/com/solvd/laba/persistence/impl/CityDAO.java

```diff
@@ -0,0 +1,217 @@
+package com.solvd.laba.persistence.impl;
+
+import com.solvd.laba.model.City;
+import com.solvd.laba.model.Road;
+import com.solvd.laba.persistence.CityRepository;
+import com.solvd.laba.persistence.ConnectionPool;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.lang.invoke.MethodHandles;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CityDAO implements CityRepository {
```

Comment:
We should stay with the same naming convention as in the interface. So it should be something like CityRepositoryImpl